src/index.tsx

```diff
@@ -13,7 +17,12 @@ const router = createBrowserRouter([
   },
```

Comment:
Shouldn't this happen perhaps on the root element? So any navigation triggers it?

Reply:
Tried that - react dom wouldn't let me :/ it's the latest and greatest in react navigation but can't say I'm impressed ...